js/coinex.js

```diff
@@ -1822,6 +1823,31 @@ module.exports = class coinex extends Exchange {
         return this.parseTrades (trades, market, since, limit);
     }
 
+    async setLeverage (leverage, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' setLeverage() requires a symbol argument');
+        }
+        if (params['position_type'] === undefined) {
+            throw new ArgumentsRequired (this.id + ' setLeverage() requires a position_type parameter that will transfer margin to the specified trading pair');
+        }
+        const market = this.market (symbol);
+        const positionType = this.safeInteger (params, 'position_type');
```

Comment:
Let's also support a marginType option as we do here:

- https://github.com/ccxt/ccxt/blob/master/js/huobi.js#L3629

```JavaScript
const defaultMarginType = this.safeString2 (this.options, 'defaultMarginType, 'marginType');
let defaultPositionType = undefined;
if (defaultMarginType === 'isolated') {
    defaultPositionType = 1;
} else if (defaultMarginType === 'cross') {
    defaultPositionType = 2;
}
const positionType = this.safeInteger (params, 'position_type', defaultPositionType);
if (positionType === undefined) {
    throw new ArgumentsRequired (this.id + ' setLeverage() requires a position_type parameter that will transfer margin to the specified trading pair');
}
```